irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -264,6 +266,18 @@ namespace iroha {
       return inserted;
     }
 
+    bool StorageImpl::insertPeer(const shared_model::interface::Peer &peer) {
+      log_->info("insert peer {}", peer.pubkey().hex());
```

Comment:
Well, I guess info is fine since you probably don't expect to see many such messages and they could be pretty important.
Also I agree about first letter issues but I don't want to use a gerund since we don't use it in other functions.